README.md

```diff
@@ -233,6 +233,15 @@ If you used Windows and decided to switch to Linux, this course should help you
 * [Bash Guide for Beginners](https://linux.die.net/Bash-Beginners-Guide/)
 * [Advanced Bash-Scripting Guide](https://linux.die.net/abs-guide/)
 
+### ZSH Useful Resources
```

Comment:
Add an empty line after this please.

Reply:
Also, let's keep it "ZSH Learning resources". And there should be a link in the Contents table to this heading.